modules/ROOT/pages/config-workflow.adoc

```diff
@@ -18,3 +18,8 @@ endif::[]
     * Register Mule runtimes
 
 After completing the steps in this workflow, you can deploy and manage applications, register APIs and use the API Exchange on an Anypoint Platform environment running on your own local network.
```

Comment:
comma before second "and":

"...register APIs, and use"

Reply:
...and use API Exchange on...
(no "the")

Reply:
Got it - thank you!